keyring/backend.py

```diff
@@ -196,7 +196,34 @@ def _load_plugins():
 
     `initialize_func` is optional, but will be invoked if callable.
     """
-    entry_points = metadata.entry_points()['keyring.backends']
+    try:
+        entry_points = metadata.entry_points()['keyring.backends']
+    except Exception:
```

Comment:
Is it possible to catch a less generic exception here?

Reply:
Yes. I will fix it